C/CImGui/bundled/wrapper/CMakeLists.txt

```diff
@@ -43,4 +43,23 @@ install(TARGETS cimgui_helper
         INCLUDES DESTINATION include
         PUBLIC_HEADER DESTINATION include)
 
+add_library(cimplot SHARED)
+
+target_sources(cimplot PRIVATE ./cimplot/cimplot.cpp)
+target_sources(cimplot PRIVATE ./cimplot/implot/implot.cpp)
+target_sources(cimplot PRIVATE ./cimplot/implot/implot_demo.cpp)
+
+target_include_directories(cimplot PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/cimplot
+    ${CMAKE_CURRENT_SOURCE_DIR}/cimplot/implot)
+
+target_link_libraries(cimplot cimgui)
```

Comment:
Since we added the imgui source code above, `cimplot` now can work without linking to cimgui.